test-clients/src/main/java/com/hedera/services/bdd/suites/contract/precompile/DynamicGasCostSuite.java

```diff
@@ -953,9 +953,7 @@ private HapiApiSpec priorityAddressIsCreate2ForStaticHapiCalls() {
 									staticCallAliasAns.get(),
 									staticCallMirrorAns.get(),
 									"Static call with mirror address should be same as call with alias");
-							assertEquals(
-									staticCallAliasAns.get().toString(16),
-									aliasAddr.get(),
+							assertEquals(new BigInteger(aliasAddr.get(), 16), staticCallAliasAns.get(),
 									"Alias should get priority over mirror address");
```

Comment:
You switched the expected and actual roles.

Reply:
I think these are the correct roles.  Alias Address is what we want in the contract call answer.

Reply:
Ok, as long as the change is intentional. Approved.